gomill_tests/sgf_tests.py

```diff
@@ -414,7 +414,7 @@ def test_serialise_transcoding(tc):
     (;FF[4]C[\xa3]CA[latin-1]GM[1]SZ[19])
     """))
     g1.get_root().set("CA", "unknown")
-    tc.assertRaisesRegexp(ValueError, "unsupported charset: \['unknown']",
+    tc.assertRaisesRegexp(ValueError, r"unsupported charset: \['unknown']",
```

Comment:
this is W605 which is helping move code to be python 3 complaint which isn't a huge deal, I can back this change (and one other similar) change out if you'd like
https://pep8.readthedocs.io/en/latest/_modules/pycodestyle.html